.github/workflows/linux-build-base.yml

```
@@ -133,49 +35,31 @@ jobs:
         shell: bash
         working-directory: velox
     steps:
-
-      - name: Get Ccache Stash
-        uses: assignUser/stash/restore@v1
+      - uses: actions/checkout@v4
         with:
-          path: '${{ env.CCACHE_DIR }}'
-          key: ccache-ubuntu-debug-default-${{ inputs.use-clang && 'clang' || 'gcc' }}
-
+          path: velox
       - name: Ensure Stash Dirs Exists
         working-directory: ${{ github.workspace }}
         run: |
           mkdir -p '${{ env.CCACHE_DIR }}'
 
-      - uses: actions/checkout@v4
-        with:
-          path: velox
-
       - name: Install Dependencies
         run: |
           source scripts/setup-ubuntu.sh && install_apt_deps
 
-      - name: Clear CCache Statistics
-        run: |
-          ccache -sz
-
-      - name: Make Debug Build
+      - name: Build Artifact
         env:
           VELOX_DEPENDENCY_SOURCE: BUNDLED
           ICU_SOURCE: SYSTEM
-          MAKEFLAGS: "NUM_THREADS=8 MAX_HIGH_MEM_JOBS=4 MAX_LINK_JOBS=3"
-          EXTRA_CMAKE_FLAGS: "-DVELOX_ENABLE_ARROW=ON -DVELOX_ENABLE_PARQUET=ON -DVELOX_ENABLE_EXAMPLES=ON"
+          MAKEFLAGS: "MAX_HIGH_MEM_JOBS=4 MAX_LINK_JOBS=3"
         run: |
-          if [[ "${USE_CLANG}" = "true" ]]; then export CC=/usr/bin/clang-15; export CXX=/usr/bin/clang++-15; fi
-          make debug
-
-      - name: CCache after
-        run: |
-          ccache -vs
-
-      - uses: assignUser/stash/save@v1
-        with:
-          path: '${{ env.CCACHE_DIR }}'
-          key: ccache-ubuntu-debug-default-${{ inputs.use-clang && 'clang' || 'gcc' }}
+          if [[ "${USE_CLANG}" = "true" ]]; then
+            export CC=/usr/bin/clang-15
+            export CXX=/usr/bin/clang++-15
+          fi
```

Sorry, since these steps are moved back into `linux-build-base.yml`, we can revert this change:

```suggestion
          if [[ "${USE_CLANG}" = "true" ]]; then export CC=/usr/bin/clang-15; export CXX=/usr/bin/clang++-15; fi
```